### DISCUSSION TITLE
2nd Moitor not working

### DISCUSSION THREAD

**Discussion by equestsolutions** [valinet/ExplorerPatcher#2570] [Q&A] [answered]
Windows 11 Pro. Microsoft® Outlook® 2019 MSO (Version 2311 Build 16.0.17029.20028) 64-bit. 

I'm able to relocate my taskbar to the top for my main display, but unable to get it to relocate for my 2nd display.

when I try to change start menu/taskbar location to Top for secondary taskbar, it will not allow me to select it. It just stays on “bottom”

Could someone help me with this?

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

Please include any other software you have installed that modifies the UI of windows.

**Comment by equestsolutions** [comment]
Edition	Windows 11 Pro
Version	23H2
Installed on	‎10/‎30/‎2023
OS build	22631.2506
Experience	Windows Feature Experience Pack 1000.22677.1000.0

ExplorerPatcher Version: 22621.2506.60.1

Move Taskbar to Top
I chose Taskbar style: 	Windows 10
Start Menu: Primary Taskbar: Top

I AM able to relocate my taskbar to the top for my main display, but UNABLE to get it to relocate for my 2nd display.
When I try to change start menu/taskbar location to Top for secondary taskbar, it will not allow me to select it. It just stays on “bottom”
Could someone help me with this?

**Comment by equestsolutions** [comment, marked as answer]
I figured it out. I just had to drag/drop the taskbar on the 2nd monitor

**Comment by pyrates999** [comment]
Please close this then if you can or mark your own response as the answer.

**Comment by equestsolutions** [comment]
solved
